src/handlers/messages.ts

```diff
@@ -241,18 +241,26 @@ export async function uploadFile(
   fileName: string,
   content_type: string
 ): Promise<string> {
-  const fileStream = fs.createReadStream(filePath)
   const accessToken = await getAccessToken(user_id)
   log.http(`Uploading ${fileName}...`)
 
+  let fd: fs.FileHandle | undefined
+  try {
+    fd = await fs.open(filePath)
+  } catch (err) {
```

Comment:
Sorry, I didn't make it clear enough.
Would you please put the whole block, including `axios.post` into a try catch with a check for the i/o error types?

Reply:
Right, seems more logical indeed. Done in https://github.com/verdigado/rocketchat2matrix/pull/32/commits/1669df6df8184dde9077901072c572aa1d60366d